de_DE/webapp/src/main/webapp/i18n/vivo_all_de_DE.properties

```diff
@@ -580,6 +580,8 @@ vis_caching_process = Was beinhaltet der Vorgang zur Zwischenspeicherung?
 vis_tools_note_two = Zu diesem Zweck haben wir eine Zwischenspeicherungslösung entwickelt, die durch die Speicherung des RDF-Modells Informationen über die Hierarchie der Organisationen speichert -- d. h. welche Publikationen welchen Organisationen zugerechnet werden.
 vis_tools_note_three = Zurzeit speichern wir diese Modelle im Hauptspeicherbereich zwischen. Der Zwischenspeicher wird (nur einmal) bei der ersten Benutzeranfrage nach einem Neustart des Servers aufgebaut. Daher wird bis zum nächsten Neustart das gleiche Modell bedient. Das bedeutet, dass die Daten in den Modellen veralten können, je nachdem, wann das Modell zum letzten Mal erstellt wurde. Das funktioniert fürs Erste gut. In kommenden Releases werden wir diese Lösung dahingehend verbessern, dass Modelle auf Festplatte gespeichert und regelmäßig aktualisiert werden.
 vis_tools_note_four = Die Modelle werden bei jedem Neustart des Servers aktualisiert. Da dies auf der Produktionsinstanz  allgemein unpraktisch ist, können AdministratorInnen stattdessen den “Zwischenspeicher aktualisieren”-Link oben verwenden, um ohne einen Neustart eine Aktualisierung durchzuführen.
+capability_map=Capability Map
```

Comment:
This should be in German, no?

Reply:
if you go down some lines the h2 tag says "Capability Map" to and the rest is in german. There is a pending ticket in the german fork where they discuss a good translation. So i think it is fine for now

Reply:
oh they resolved it, i will change it now. it will be "Forschungsnetz"